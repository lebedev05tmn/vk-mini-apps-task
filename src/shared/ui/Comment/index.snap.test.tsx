import { Comment } from "./index";

const ReactTestRenderer = require("react-test-renderer");

describe("Comments render", () => {
  test("Should Comment is not Sub", () => {
    const tree = ReactTestRenderer.create(
      <Comment
        text={"Hello world"}
        by={"author"}
        time={"12345"}
        onClick={() => console.log("hello world")}
        index={1}
        haveSub={false}
        isSub={false}
      />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test("Should Comment is Sub", () => {
    const tree = ReactTestRenderer.create(
      <Comment
        text={"Hello world"}
        by={"author"}
        time={"12345"}
        onClick={() => console.log("hello world")}
        index={1}
        haveSub={false}
        isSub={true}
      />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
