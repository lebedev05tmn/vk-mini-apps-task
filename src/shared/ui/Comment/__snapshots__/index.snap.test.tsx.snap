// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments render Should Comment is Sub 1`] = `
<div
  className="vkuiCard undefined undefined"
>
  <span
    className="vkuiTypography vkuiTypography--normalize vkuiTitle--level-1"
  >
    1
     Автор: 
    author
  </span>
  <span
    className="vkuiTypography vkuiParagraph"
  >
    Hello world
  </span>
  <span
    className="vkuiTypography vkuiParagraph"
  >
    12345
  </span>
</div>
`;

exports[`Comments render Should Comment is not Sub 1`] = `
<div
  className="vkuiCard undefined false"
>
  <span
    className="vkuiTypography vkuiTypography--normalize vkuiTitle--level-1"
  >
    1
     Автор: 
    author
  </span>
  <span
    className="vkuiTypography vkuiParagraph"
  >
    Hello world
  </span>
  <span
    className="vkuiTypography vkuiParagraph"
  >
    12345
  </span>
</div>
`;
